thrift/src/main/thrift/entities/person.thrift

```diff
@@ -0,0 +1,5 @@
+namespace java com.gu.contententity.thrift.entity.person
+
+struct Person {
+  1: required string fullName
```

Comment:
I think it is fine for the time being but I think it will be great to have in the future:

`firstname`
`lastname`
`birthdate`
`birthplace`
`deathdate`